purchase_partial_invoicing/__openerp__.py

```diff
@@ -23,7 +23,8 @@
     'name': "Purchase partial invoicing",
     'version': '0.1',
     'category': 'Purchase Management',
-    'author': "Agile Business Group,Odoo Community Association (OCA)",
+    'author':
```

Comment:
You can split it this way to avoid this strange layout:
```
'author': "Agile Business Group,"
          "ACSONE SA/NV,"
          "Odoo Community Association (OCA)",
```